app/pages/SearchResultsPage/SearchResultsPage.tsx

```diff
@@ -102,14 +102,14 @@ export const SearchResultsPage = () => {
     setSearchState({ ...nonQuerySearchParams, query: translatedQuery });
   }, [translatedQuery, nonQuerySearchParams]);
 
-  if (translatedQuery === null || searchState === null) {
-    return null;
+  if (translatedQuery === null || searchState === null || userLocation === null) {
```

Comment:
Changes in this file fix bug from search by location PR https://github.com/ShelterTechSF/askdarcel-web/pull/1339